### ISSUE TITLE
^5.4.7-beta.1 - Error: NativeViewGestureHandler must be used as a descendant of GestureHandlerRootView. Otherwise the gestures will not be recognized. See https://docs.swmansion.com/react-native-gesture-handler/docs/installation for more details.

### ISSUE THREAD

**Issue by sharathbangera6** [hossein-zare/react-native-dropdown-picker#725]
I am getting following error when i use ^5.4.7-beta.1 Version -   NativeViewGestureHandler must be used as a descendant of GestureHandlerRootView. Otherwise, the gestures will not be recognized. See https://docs.swmansion.com/react-native-gesture-handler/docs/installation for more details.

Can you please import GestureHandlerRootView from react-native-gesture-handler and Add it to the route of your component. 

I edit Picker.js and added below code to route of this component and it works. Can you incorporate the same and update ? we will upgrade the package. 

```
<GestureHandlerRootView style={{ flex: 1 }}> ... </GestureHandlerRootView>

**Comment by dominictobias** [comment]
Yep same issue too, my app is already wrapped in `GestureHandlerRootView` 🤔

**Comment by dominictobias** [comment]
This issue happens when yarn doesn't properly dedupe deps. Run `npm ls react-native-gesture-handler`

I had `react-native-gesture-handler` duped in `react-native-dropdown-picker` even though all my deps were using the same version and yarn claims it de-dupes.

In the end I just deleted `yarn.lock` and `node_modules` and reinstalled and de-duping worked properly

```
myproject
├─┬ @react-navigation/drawer@6.6.6
│ └── react-native-gesture-handler@2.13.4 deduped
├─┬ react-native-draggable-flatlist@4.0.1
│ └── react-native-gesture-handler@2.13.4 deduped
├─┬ react-native-dropdown-picker@5.4.7-beta.1
│ └── react-native-gesture-handler@2.13.4 deduped // before it wasn't deduped
└── react-native-gesture-handler@2.13.4
```

**Comment by MussadiqAli** [comment]
Just remove node_modules and yarn.lock, and yarn install. Problem is solved

**Comment by warmfire540** [comment]
got this working on a fork, you can try and see if it works for you

```bash
yarn add react-native-dropdown-picker@https://github.com/curious-cat-consulting/react-native-dropdown-picker#5.5.0
```

i took the 5.4.7 beta and fixed a peer dep issue for expo
